### DISCUSSION TITLE
Adding New Render Hooks on Auth Pages

### DISCUSSION THREAD

**Discussion by SuperUserDo-Nikola** [filamentphp/filament#13793] [Ideas]
Hi fellas, 

I run into small bit of issue, not be able to found right hook for my case. And in my opinion it would be useful for other developers to have it also. It will make designing Auth Pages a big easier. 

I did a workaround with inserting HTML via JS...

But let's go straight to case: 

File: vendor/filament/filament/resources/views/components/layout/simple.blade.php. 
Possible Solution: To add Render Hook right after element with class "fi-simple-main-ctn" and before <main>, and one hook after main element.

Code: 
```
<div class="fi-simple-main-ctn flex w-full flex-grow items-center justify-center" >
            // Here would be hook
            <main
                @class([
                    'fi-simple-main my-16 w-full bg-white px-6 py-12 shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10 sm:rounded-xl sm:px-12',
                    match ($maxWidth ?? null) {
                        MaxWidth::ExtraSmall, 'xs' => 'sm:max-w-xs',
                        MaxWidth::Small, 'sm' => 'sm:max-w-sm',
                        MaxWidth::Medium, 'md' => 'sm:max-w-md',
                        MaxWidth::ExtraLarge, 'xl' => 'sm:max-w-xl',
                        MaxWidth::TwoExtraLarge, '2xl' => 'sm:max-w-2xl',
                        MaxWidth::ThreeExtraLarge, '3xl' => 'sm:max-w-3xl',
                        MaxWidth::FourExtraLarge, '4xl' => 'sm:max-w-4xl',
                        MaxWidth::FiveExtraLarge, '5xl' => 'sm:max-w-5xl',
                        MaxWidth::SixExtraLarge, '6xl' => 'sm:max-w-6xl',
                        MaxWidth::SevenExtraLarge, '7xl' => 'sm:max-w-7xl',
                        default => 'sm:max-w-lg',
                    },
                ])
            >
                {{ $slot }}
            </main>
           // Second hook
</div>
```
Example in photo, and would be same case for register. (I painted company info for safety purposes)
![example](https://github.com/user-attachments/assets/33f4120f-eb61-43c1-86cd-7f9303cda313)

**Comment by rvzug** [comment]
Aren't 
- ```PanelsRenderHook::AUTH_LOGIN_FORM_BEFORE```
- ```PanelsRenderHook::AUTH_LOGIN_FORM_AFTER```

not those hooks?

See: https://filamentphp.com/docs/3.x/support/render-hooks#panel-builder-render-hooks

**Reply by SuperUserDo-Nikola** [reply]
Nah, those are within login "card", I was referring to ones outside of main login "card".
